exonum-java-binding-core/rust/src/runtime/cmd.rs

```diff
@@ -58,16 +50,18 @@ impl CommandExtension for GenerateNodeConfig {
     }
 
     fn execute(&self, mut context: Context) -> Result<Context, failure::Error> {
-        let user_parameters = context.arg_multiple(EJB_JVM_ARGUMENTS).unwrap_or_default();
+        let user_parameters = Vec::new();
```

Comment:
Are `user_parameters` and `jvm_debug_socket` below needed at this stage (generate node config)?

Reply:
Replied below. Going to extract these fields to other structure.